### PR TITLE
There are something new in Chinese translation.

### PR DESCRIPTION
**Pull request by 3442853561**
一些中文翻译的新玩意儿

### CODE REVIEW

File: zh-CN/downloads.html [hunk 1]
```diff
@@ -9,12 +9,11 @@
     <div class="row install">
       <div class="col-md-4 side-header">
         <h2 id="stable">{{ site.stable }}&nbsp;</h2>
-        <h3>{{ site.stable_date | date: "%B %-d, %Y" }}</h3>
+        <h3>{{ site.stable_date | date: "%Y 年 %m 月 %-d 日" }}</h3>
```
Comment (lifthrasiir) [inline zh-CN/downloads.html]: Shouldn't this be `%Y年%-m月%-d日`, like "2017年1月1日" or "2017年1月15日"? That is, no leading zero, and no space inbetween. Also, `site.beta_date` remains untranslated.

(Tip: If you want weekdays as well, you can make use of [arrays](https://github.com/rust-lang/rust-www/blob/74af2f75c9dc9439332d9a34fa7702570a86cc92/pt-BR/downloads.html#L9-L11) as previously used by `pt-BR`. Liquid's support of array is at best strange, though.)

File: zh-CN/team.md [hunk 2]
```diff
@@ -213,7 +212,7 @@ the [Rust security disclosure process](security.html).
     {% assign sitename = "github" %}
   {% endif %}
   {% assign site = page.sites[sitename] %}
-  <li class="person {% if team.lead and team.lead == nick %}lead{% endif %}">
+  <li class="person {% if team.lead and team.lead == nick %}负责人{% endif %}">
```
Comment (lifthrasiir) [inline zh-CN/team.md]: This is a CSS class and should not be translated.

File: zh-CN/downloads.html (hunk 1 above)
Comment (3442853561) [inline zh-CN/downloads.html]: In Chinese there need a space between （number or letter ）and （chinese words）

File: zh-CN/team.md (hunk 2 above)
Comment (3442853561) [inline zh-CN/team.md]: It has been modified

File: zh-CN/downloads.html (hunk 1 above)
Comment (3442853561) [inline zh-CN/downloads.html]: I don't need weekdays as well,%m mean month,
and I don't know want mean about `site.beta_date`
@lifthrasiir

File: zh-CN/team.md (hunk 2 above)
Comment (3442853561) [inline zh-CN/team.md]: @lifthrasiir fixed

File: zh-CN/downloads.html (hunk 1 above)
Comment (lifthrasiir) [inline zh-CN/downloads.html]: @3442853561 My impression from various Chinese websites was that there are no spaces between numbers and Chinese letters, but if a space is needed, okay with that.

`%m` will always display two digits for month. `%Y 年 %m 月 %-d 日` will give you "2017 年 01 月 1 日" for example, which looks strange.

On `site.beta_date`, there are two dates for the latest stable release and the upcoming stable release (which is currently at beta). You will want to translate both.

Comment (3442853561) [inline zh-CN/downloads.html]: Thanks！I will fix it.

Comment (3442853561) [inline zh-CN/downloads.html]: @lifthrasiir THX,I fixed it.

**Comment by alexcrichton** [comment]
Thanks!

**Pull request merged**